recipes/cotila/all/conanfile.py

```diff
@@ -0,0 +1,37 @@
+from conans import ConanFile, CMake, tools
+
+required_conan_version = ">=1.33.0"
+
+
+class CotilaConan(ConanFile):
+    name = "cotila"
+    description = "A compile time linear algebra system"
+    homepage = "https://github.com/calebzulawski/cotila"
+    topics = ("c++", "math", "linear algebra", "compile-time")
+    license = "Apache-2.0"
+    url = "https://github.com/conan-io/conan-center-index"
+    settings = "os", "compiler", "build_type", "arch"
+    generators = "cmake"
+    no_copy_source = True
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    @property
+    def _min_cppstd(self):
+        return "17"
+
+    def package_id(self):
+        self.info.header_only()
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version], strip_root=True,
+                  destination=self._source_subfolder)
+
+    def package(self):
+        self.copy(pattern="LICENSE", dst="licenses", src=self._source_subfolder)
+        self.copy(pattern="include/*", dst="include", src=self._source_subfolder)
+
+    def package_info(self):
+        self.cpp_info.includedirs = ["include/"]
```

Comment:
- `include` is added by default.
- https://github.com/calebzulawski/cotila/blob/2eeb66bb4427cf0b8155c02b9e5a632d0a059f45/CMakeLists.txt#L36: set the names for the `cmake_find_package(_multi)` explicitly.
```suggestion
        self.copy(pattern="include/*", dst="include", src=self._source_subfolder) 

        def package_info(self):
            self.cpp_info.names["cmake_find_package"] = "cotila"
            self.cpp_info.names["cmake_find_package_multi"] = "cotila"
```